ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
@gtirloni, those look like different points of failure to me - it's successfully resolving the 'infusion' module in the container case. Your runs look like they aren't successful installs (from a 'universal can be run' standpoint) - does `node gpii` work?